desktop_object.py

```diff
@@ -131,6 +127,14 @@ def _get_names(self):
                                               self._localized_names[locale])
         return name_string
 
+    def _get_categories(self):
+        category = self._data['linkCategory']
+        if category == 'Education and Health':
+            category = 'EducationAndHealth'
+        # Start with a valid category per the freedesktop spec,
```

Comment:
Any reason to always add `Network` here, or to change the code path for links at all? Note that they're ignored by gnome-software.